### PR TITLE
Site 6736 button polish

### PR DESCRIPTION
**Pull request by Jiert**
This is a work in progress, but I wanted to create the PR so people could comment on it

### CODE REVIEW

**Comment by cohitre** [comment]
lgtm

**Comment by cohitre** [comment]
squash, update changelog, version

File: app/components/GroupCircled.js [hunk 1]
```diff
@@ -19,25 +19,31 @@ module.exports = defineSvg('GroupCircled', {
...
+    return (
```
Comment (tildedave) [inline app/components/GroupCircled.js]: possible to use defineSvg here?

**Comment by tildedave** [comment]
looks like this will be a complete viewbox change on this icon so we probably want to bump to 6.0.  really pumped these are props on the existing icon

File: app/components/GroupCircled.js (hunk 1 above)
Comment (Jiert) [inline app/components/GroupCircled.js]: @tildedave already doing it :point_up:

Comment (tildedave) [inline app/components/GroupCircled.js]: coo

File: app/stylesheets/support/_colors.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
 $tilt-blue: #00A0D8;
 $tilt-light-gray: #D9D9D9;
 $tilt-gray: #999999;
+$tilt-blue-hover: #00B6F6;
```
Comment (cohitre) [inline app/stylesheets/support/_colors.scss]: unused?

**Comment by cohitre** [comment]
lgtm

**Comment by cohitre** [comment]
needs squash, package.json version update and cutting the GH tag

File: app/stylesheets/support/_colors.scss (hunk 2 above)
Comment (Jiert) [inline app/stylesheets/support/_colors.scss]: @cohitre good catch :hocho:

**Pull request merged**